application/organisation-affiliations-data-manager/oa_service.py

```diff
@@ -0,0 +1,44 @@
+import boto3
```

Comment:
Name of file/module - org_affiliations_service ?
Exception handling now or later